carsus/io/tests/test_cmfgen.py

```diff
@@ -0,0 +1,105 @@
+import os
+import glob
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+from carsus.io.cmfgen import (CMFGENEnergyLevelsParser,
+                              CMFGENOscillatorStrengthsParser,
+                              CMFGENCollisionalDataParser,
+                              CMFGENPhotoionizationCrossSectionParser,
+                             )
+
+
+REF_DATA_DIR = os.getenv('CARSUS_REFDATA')
```

Comment:
that should be a commandline option like in `tardis`

Reply:
I will try.

Reply:
It's working https://travis-ci.org/tardis-sn/carsus/jobs/553922387